src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
```

Comment:
Consider using `describe` to group related tests for better organization and readability. For example: `describe('splitInteger function', () => { /* your tests here */ })`